lib/ruby-prof/printers/flat_printer.rb

```diff
@@ -29,8 +29,9 @@ def print_methods(thread)
 
       sum = 0
       methods.each do |method|
-        self_percent = (method.self_time / total_time) * 100
-        next if self_percent < min_percent
+        percent = (method.send(filter_by) / total_time) * 100
```

Comment:
I noticed an interesting thing here. I defaulted `filter_by` to `:self_time`, which is fine for this printer. But, for example: https://github.com/ruby-prof/ruby-prof/blob/e6c7d9b605e327f4852104ac2dcd989ad80310f9/lib/ruby-prof/printers/graph_printer.rb#L55-L56

The default is actually filtering by `:total_time`! This is fixable: I can just set a different default per printer class, but it is a little confusing.